packages/chart/src/Pie.ts

```diff
@@ -411,3 +415,4 @@ Pie.prototype.publish("innerRadius", 0, "number", "Sets inner pie hole radius as
 Pie.prototype.publish("minOuterRadius", 20, "number", "Minimum outer radius (pixels)");
 Pie.prototype.publish("startAngle", 0, "number", "Starting angle of the first (and largest) wedge (degrees)");
 Pie.prototype.publish("labelHeight", 12, "number", "Font size of labels (pixels)");
+Pie.prototype.publish("percentageDecimals", 1, "number", "Number of decimal places to display while showing series percentage", null, {disable: w => !w.showSeriesPercentage()});
```

Comment:
Can you change this so it matches what we do with the tooltip values:  https://github.com/hpcc-systems/Visualization/blob/master/packages/api/src/ITooltip.ts#L154

Reply:
Hmm, yeah that makes sense... I'm on it